Query/EncloseNames.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * Hoa
+ *
+ *
+ * @license
+ *
+ * New BSD License
+ *
+ * Copyright © 2007-2015, Hoa community. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the Hoa nor the names of its contributors may be
+ *       used to endorse or promote products derived from this software without
+ *       specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS AND CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+namespace Hoa\Database\Query;
+
+/**
+ * Abstract Class \Hoa\Database\Query\EncloseNames.
+ *
+ * Enclose names feature.
+ *
+ * @copyright  Copyright © 2007-2015 Hoa community
+ * @license    New BSD License
+ */
+abstract class EncloseNames
+{
+    /**
+     * State of the enclose names feature.
+     *
+     * @var bool
+     */
+    protected $enableEnclose = false;
+
+    /**
+     * Enclose open symbol.
+     *
+     * @var string
+     */
+    protected $openSymbol    = '"';
+
+    /**
+     * Enclose close symbol.
+     *
+     * @var string
+     */
+    protected $closeSymbol   = '"';
+
+
+
+    /**
+     * Set enclose symbols.
+     *
+     * @param  string  $openSymbol     Open symbol.
+     * @param  string  $closeSymbol    Close symbol.
+     * @return  static
+     */
+    public function setEncloseSymbol($openSymbol, $closeSymbol = null)
+    {
+        $this->openSymbol  = $openSymbol;
+        $this->closeSymbol = (null === $closeSymbol)
```

Comment:
Same here, CS:
```php
$this->_closeSymbol =
    (null === $closeSymbol)
        ? $openSymbol
        : $closeSymbol;
```
or everything inlined since it might not be longer than 80 columns.